README.md

```diff
@@ -1836,8 +1836,8 @@ Gets the value at path of object.
   // output: 3
   
   // Native
-  const get = (obj, path, defaultValue) => path.split(".")
-  .reduce((a, c) => (a && a[c] ? a[c] : (defaultValue || null)), obj)
+  const get = (obj, path, defaultValue) => String.prototype.split.call(path, /,?[\[\]]?\.?/)
```

Comment:
`[\[` better be replaced with `[[` since escaping of `[` in a `[...]` construct is useless

Reply:
You haven't tested it on path keys longer than 1 character, have you? (@clujin)